### DISCUSSION TITLE
Grapesjs with Metamask Api

### DISCUSSION THREAD

**Discussion by kmlcnclk** [GrapesJS/grapesjs#5406] [Q&A]
I am developing a project which user can create own nft marketplace design using Grapesjs. Other people can buy a nft on that design. I am gettiing an error like at image when i droped the first time nft item to the page,

<img width="461" alt="Screenshot 2023-09-21 at 09 34 02" src="https://github.com/GrapesJS/grapesjs/assets/56474929/8e1912e7-6b5a-4114-97d5-2e4f22da04d4">

When i dropped the second time nft item to the page, It is happening like in the image. Grapesjs didn't find Chrome Extension. Why these problems happening ? Can you help me ?

<img width="487" alt="Screenshot 2023-09-21 at 09 43 50" src="https://github.com/GrapesJS/grapesjs/assets/56474929/d061d16e-9099-4e0b-8421-cdc9b0a34157">

**Comment by artf** [comment]
Please provide a [reproducible demo](https://jsfiddle.net/szLp8h4n).
